plugins/task-plugin/src/machine/attach-client.ts

```diff
@@ -0,0 +1,54 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import * as ws from 'ws';
+import { injectable, inject } from 'inversify';
+import { CheWorkspaceClient } from '../che-workspace/che-workspace-client';
+
+const ReconnectingWebSocket = require('reconnecting-websocket');
```

Comment:
from where this dependency is coming from ? (I don't see it in package.json) and there is no CQ

Reply:
I'm removing it from my PR, but btw, we use it at least [here](https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-terminal/src/browser/server-definition/remote-connection.ts#L16), [here](https://github.com/eclipse/che-theia-task-plugin/blob/master/che-theia-task-extension/src/browser/che-terminal-widget.ts#L29)  and [here](https://github.com/eclipse/che-theia-task-plugin/blob/master/che-theia-task-extension/src/node/json-rpc-proxy-provider.ts#L14)
So, should we remove it from all these places?